src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -498,11 +498,11 @@ class NwbRecordingExtractor(BaseRecording, _BaseNWBExtractor):
     >>>
     >>> # get s3 path
     >>> dandiset_id, filepath = "101116", "sub-001/sub-001_ecephys.nwb"
-    >>> with DandiAPIClient("https://api-staging.dandiarchive.org/api") as client:
```

Comment:
oh, actually we may need to keep this, since I guess the sample data is in staging

Reply:
I guess we coudl jsut select other dataset that is not in staging area which I think would be better. What do you think?

Reply:
@h-mayorquin so should we pick another dataset?

Reply:
Yes.I think it will be better if the docstring does not have that complication.